Buildings/Components/RoofBase.gd

```diff
@@ -0,0 +1,16 @@
+extends Node3D
+class_name RoofBase
+
+var addon_layer: GeoFeatureLayer
+var addon_object: PackedScene
```

Comment:
I assume this might become a dictionary mapping some kind of ID to PackedScenes in the future, maybe that can be changed (and added to the config) right away?

Reply:
[80b052a](https://github.com/boku-ilen/landscapelab/pull/338/commits/80b052a5b32bb7b72d565e3b659aadf89ef521cc)